CMakeLists.txt

```
@@ -14,7 +14,7 @@ set_from_environment(Fallout4Path)
 
 project(
 	Buffout4
-	VERSION 1.24.5
+	VERSION 1.24.6
```

```suggestion
	VERSION 1.25.0
```
New features should increment the minor version